engines/dfc_provider/spec/requests/supplied_products_spec.rb

```diff
@@ -82,6 +82,21 @@
         end
       end
 
+      # TODO fix test
+      response "401", "unauthorized" do
```

Comment:
I am not sure how to trigger a 401 here. If we try to add a product with a user that is not "owner" or "manager" of the enterprise we get a 404 due to the `check_enterprise` here : https://github.com/openfoodfoundation/openfoodnetwork/blob/af9f07f496052ff77b033bd8e6e49e8803e89f0f/engines/dfc_provider/app/controllers/dfc_provider/application_controller.rb#L23
So I am wondering if we even need the `authorize!` here (same for the update action) ?